apps/public-reference/pages/applications/household/add-members.tsx

```diff
@@ -73,25 +73,27 @@ export default () => {
           </h2>
         </div>
 
-        <div className="form-card__pager-row">
-          <form onSubmit={handleSubmit(onSubmit)}>
-            <div className={errors.householdSize ? "mb-8" : ""}>
-              <HouseholdSizeField
-                listing={listing}
-                householdSize={application.householdSize}
-                validate={true}
-                register={register}
-                error={errors.householdSize}
-                clearErrors={clearErrors}
-                assistanceUrl={t("application.household.assistanceUrl")}
-              />
-            </div>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <div className={errors.householdSize ? "" : ""}>
```

Comment:
Is this intentional? As written, we're not adding classes either way, should just be `<div>`...

Reply:
@bencpeters Huh, I don't know how changes to this page ended up in this PR. Maybe it's something Jesse worked on. I will double-check.